source/Ship.cpp

```diff
@@ -2052,9 +2052,9 @@ bool Ship::IsEnteringHyperspace() const
 
 
 
-bool Ship::IsHyperspacing() const
+int Ship::IsHyperspacing() const
```

Comment:
should probablz be renamed to GetHyperspaceCount() as the is implies a bool return